src/webln/NostrWeblnProvider.ts

```diff
@@ -213,6 +236,40 @@ export class NostrWebLNProvider {
       });
     });
   }
+
+  initNWC(provider: string, options: { name: string, returnTo?: string }) {
+    const height = 600;
```

Comment:
I have a feeling this is not going to work well on mobile (not just the responsive part, but launching popups). Is there a reason you didn't go with the callback idea?

Reply:
I guess this is what returnTo is for? in my opinion if it is passed, it should open the url in the current window rather than launching the popup

Reply:
This is for browser contexts.
and this is a helper, if somebody wants it differently they can do it differently. 
most OAuth apps typically open a popup I also like this better than completely navigating the user away from the website.